tests/unit/services/test_opencodelists.py

```diff
@@ -4,14 +4,44 @@
 from services import opencodelists
 
 
-def test_fetch_returns_codelists(mock_codelists_response):
+@pytest.fixture
+def codelists(responses):
+    responses.add(
+        method="GET",
+        url=opencodelists.LIST_URL,
```

Comment:
I wondered by `LIST_URL` had been pulled out of the `fetch` function, this is a nice trick 👍